databricks/koalas/tests/test_indexes.py

```diff
@@ -75,7 +75,12 @@ def test_to_series(self):
         kidx = self.kdf.index
 
         self.assert_eq(kidx.to_series(), pidx.to_series())
-        self.assert_eq(kidx.to_series(name="a"), pidx.to_series(name="a"))
+        self.assert_eq(repr(kidx.to_series(name="a")), repr(pidx.to_series(name="a")))
+
+        pidx.name = "Koalas"
+        kidx.name = "Koalas"
+        self.assert_eq(repr(kidx.to_series()), repr(pidx.to_series()))
+        self.assert_eq(repr(kidx.to_series(name="a")), repr(pidx.to_series(name="a")))
```

Comment:
Could you add some more tests with tupled names?

- `ks.Index([1,2,3], name=('x', 'a')).to_series()`
- `ks.Index([1,2,3]).to_series(name=('x', 'a'))`

if those are valid in pandas as well?

Reply:
@ueshin Sure, I'm going to add it. Thanks!

Reply:
> What about the second one to use a tupled name as a parameter?

Sounds good. I just addressed.

Thanks for the comment !